README.md

```diff
@@ -19,18 +19,14 @@ Since both the versions use `kotlin` for some or all of their classes, following
 - After checking out to `develop` branch as mentioned above, simply import the project in Android Studio.
 
 #### v2+
-- This version requires the [Kotlin SDK](https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK) for Rocket.Chat. Clone the Kotlin SDK in the **same directory** as the android repository by running `git clone https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK.git`. Make sure that the android repository and the kotlin sdk have the same immediate parent directory.
-- First, a build is required for the SDK. Change your directory to the SDK directory by running `cd Rocket.Chat.Kotlin.SDK/` in your terminal. Any of the following approaches can be followed to successfully build the SDK.
-    - **Command Line** - (Within the kotlin SDK directory) Run `./gradlew clean && ./gradlew assemble` to successfully build the project.
-    - **Android Studio** - Import the project in Android Studio. Go to `Build > Make Project` to build the SDK successfully.
-After following the above methods, follow the following steps in your terminal window:
+- This version requires the [Kotlin SDK](https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK) for Rocket.Chat. Clone the Kotlin SDK in by running `git clone https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK.git`.
+- First, a build is required for the SDK, so that required jar files are generated. Any of the following approaches can be followed to successfully build the SDK.
+    - **Manually** - Change your directory to the SDK directory by running `cd Rocket.Chat.Kotlin.SDK/` in your terminal. (Within the kotlin SDK directory) Run `./gradlew clean && ./gradlew assemble` to successfully build the project. Now copy `common*.jar` from `Rocket.Chat.Kotlin.SDK/common/build/libs` and `core*.jar`  form `Rocket.Chat.Kotlin.SDK/core/build/libs` to `Rocket.Chat.Android/app/libs` . There will no `libs` directory in `Rocket.Chat.Android/app`, you will have to manually create it.
+    - **Using Script(recommeded)** - Make sure that the android repository and the kotlin sdk have the same immediate parent directory. Change the current directory to `Rocket.Chat.Android/app` and run the `build-sdk.sh` which will result in creating of the required jar file `core*.jar` and `common*.jar` in `Rocket.Chat.Android/app/libs`,by the following steps in your terminal window:
 ```
-cd ..
-cd Rocket.Chat.Android/app/libs
-ls
+cd Rocket.Chat.Android/app
+./build-sdk.sh
 ```
-Two `jar` files will be found in this directory (the `common` and `core` jar files), this indicates that the SDK was built correctly.
-- After the SDK has been built successfully, import the project in Android Studio and build it by following `Build > Make Project`.
```

Comment:
Just move this line above line number 32

Reply:
If the build.gradle(app) already run the build-sdk.sh then we only need to import the Rocket.Chat.Android to start the project. Are you able to get it working by doing so? 

Please check it on your system by removing both the directories - Rocket.Chat.Android and Rocket.Chat.Kotlin.SDK and again set up your environment by importing only Rocket.Chat.Android. 

You can check if it has successfully imported(even though the jar files are present in libs directory they don't work in my case when I set up my environment by importing the project) by trying to call any API method of SDK inside the project.

Reply:
Yeah that seems like an issue, the sdk needs to be built manually in order to work correctly. It works fine after doing a gradle sync though!